spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -29,7 +29,8 @@
         )
         sign_in user
 
-        post :logout, SAMLResponse: 'PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4' \
+        post :logout, params: {
+SAMLResponse: 'PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4' \
```

Comment:
indentation seems wrong here? I'd expect us to line up the strings on the next few lines?